tests/test_assets/genes_biodomains/input/biodomains_test_input_bad_should_fail.csv

```diff
@@ -1,10 +1,6 @@
-biodomain,abbr,label,color,go_id,go_terms,ensembl_gene_id
-,Au,Autophagy [Au],#9931fd,GO:0006914,autophagy,ENSG00000161011
-Autophagy,Au,Autophagy [Au],#9931fd,GO:0016236,,ENSG00000161011
-Autophagy,Au,Autophagy [Au],#9931fd,GO:0000422,autophagy of mitochondrion,
-,,,,,,
-,,,,,,
-,,,,,,
-,,,,,,
-,,,,,,
-,,,,,,
\ No newline at end of file
+biodomain,abbr,label,color,go_id,go_terms,n_symbol,symbol,ensembl_gene_id

```

Comment:
This file has no new test cases and is unchanged except for adding 2 new columns from the new biodomains file.